src/m_misc.c

```diff
@@ -2297,7 +2308,14 @@ default_t defaults[] = {
     {.s = "soundfonts"},
 #endif
     {0}, string, ss_none, wad_no,
-    "FluidSynth soundfont path"
+    "FluidSynth soundfont directory"
```

Comment:
We don't need the `WOOFSOUNDFONT` macro anymore. I'll just add the three directories that come into question across different Linux distribution in the `!_WIN32` case.